### ISSUE TITLE
Feature to select a chunk

### ISSUE THREAD

**Issue by rdadkins** [toolbox4minecraft/amidst#202]
Add a feature to click on a chunk to see its corner boundaries, biome type, and any other viable information. This can be useful if a user is looking for slime chunks and can click one to easily map out corners.

**Comment by stefandollase** [comment]
There is a layer for slime-chunks. The size of each pink square is exactly one chunk. One chunk can contain multiple biome types, so we would have to display multiple biome types per chunk.

The boundaries can be determined by using the maximum zoom and hovering the mouse over each of the corners. But I can see that this is tedious.

Clicking on a chunk to select it will probably conflict with the ability to select world icons by clicking on them. Thus, we need a mechanism to tell Amidst whether it should select the chunk or the world icon. This might work by using shift to force Amidst to select the chunk instead of the world icon.

This is mainly a convenience feature request, right? I can add it to the list of requested features, but it will probably take some time until we get to this.

Are there more information that should be displayed?

**Comment by CubeTheThird** [comment]
Perhaps instead of clicking, you could add the chunk boundaries to the coordinates displayed with mouse hovering. Perhaps a toggle to enable this, and have it turned off by default? Just a thought.

**Comment by stefandollase** [comment]
@CubeTheThird
While this sounds like a good idea at first, I think there will be some usability issues with that: I imagine the user will use this to find e.g. the border of the slime farm that he wants to build in his world. Thus, he will have Amidst and Minecraft open at the same time. Amidst should display the coordinates and he wants to search these coordinates in his world. However, this does not work properly with the mouse hovering, because as soon as he switches back to Minecraft, the mouse will no longer hover over Amidst and thus no information will be displayed, which forces him to constantly switch between the windows to find the desired location. Am I missing something?

**Comment by CubeTheThird** [comment]
This is true, however the same could then be said for any arbitrary coordinate in general. If that specific use-case is the ideal goal of this feature suggestion, I would recommend changing it to "select a coordinate" instead of "select a chunk".

**Comment by stefandollase** [comment]
Well, you are right, I might have over-interpreted the OP to this specific use-case. Indeed, the same problem exists for a single coordinate. I guess this can be solved by displaying the chunk border as well as the exact coordinate that was clicked by the user?

**Comment by CubeTheThird** [comment]
IMO that would make the most sense.

**Comment by rdadkins** [comment]
@stefandollase sorry for the miscommunication, I never got notifications about this. Originally when I thought of this feature, you would click on area on the map and the coordinates could get pinned somewhere on the window (similar to hovering on a map shows coordinates and biome type). This could be pinned (probably) in the area where coordinates display themselves in the same fashion how the seed displays itself as well as the world type.

**Comment by stefandollase** [comment]
@rdadkins Did you read the discussion above? The proposed solution is to shift-click somewhere one the map to pin a location. The pinned location is then displayed in the top right corner. It includes the exact coordinates, the biome type as well as the chunk boundaries. Does this work for you?

**Comment by rdadkins** [comment]
@stefandollase Yes, that would work perfectly
